projects/aws/eks-anywhere-packages/Makefile

```diff
@@ -4,21 +4,35 @@ GOLANG_VERSION?="1.17"
 
 REPO=eks-anywhere-packages
 REPO_OWNER=aws
+
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
+GO_MOD_PATHS=. ecrtokenrefresher
+
 IMAGE_COMPONENT=eks-anywhere-packages
-IMAGE_NAMES=$(IMAGE_COMPONENT)
+ECRTOKENREFRESHER_IMAGE_COMPONENT=eks-anywhere-packages/ecrtokenrefresher
```

Comment:
This is an add repository name from the normal pattern. I would expect just `ecrtokenrefresher` or `ecr-token-refresher`

Reply:
Updated